### DISCUSSION TITLE
ITead “SONOFF Zigbee Bridge Pro” (ZBBridge-P) new CC2652P and ESP32 based Zigbee Gateway to WiFi Bridge

### DISCUSSION THREAD

**Discussion by Hedda** [zigpy/zigpy#885] [General]
ITead Sonoff ZBBridge-P v1.1 will clearly use new CCC2652P (CC2652P1/CC2652P1F) based “SM-031 v1.1” Zigbee radio-module from CoolKit:

https://fccid.io/2APN5ZBBRIDGEP

UPDATE! ZBBridge-P (a.k.a. "ZB Bridge-P") has now been released as "SONOFF Zigbee Bridge Pro":

https://itead.cc/product/sonoff-zigbee-bridge-pro/

Found by @digiblur

https://www.digiblur.com/2021/12/new-sonoff-zigbee-bridge-and-sensors.html

Posted more details in Home Assistant community forum:

https://community.home-assistant.io/t/itead-sonoff-zbbridge-p-coming-soon-a-new-cc2652-and-esp32-based-sonoff-zigbee-bridge-and-sensors-from-itead-leaked-via-fcc/378924

Very sad that they did not add an Ethernet port. Instead, they simply replaced only the radio-module and the ESP8266/ESP8285 with ESP32:

![image](https://user-images.githubusercontent.com/6320001/149169002-8f765532-351d-499a-b10e-8d6196d3f7a8.png)

![image](https://user-images.githubusercontent.com/6320001/149169022-07eb7bb4-98a4-4cd8-b18d-0fe5ec241dd5.png)

![image](https://user-images.githubusercontent.com/6320001/149169041-9125c687-b686-49d0-af7a-4bf04b60dac5.png)

![image](https://user-images.githubusercontent.com/6320001/149169056-1680942b-b23c-4c8e-ba67-84c1227313a7.png)

![image](https://user-images.githubusercontent.com/6320001/149169077-adc48e08-c162-4935-b223-2b60766a246b.png)

**Comment by MattWestb** [comment]
I think it was not **only** replacing the Zigbee module.
It have extra ram and flash plus RTC with battery backup.
And the SM-031 looks much better in the RF part (counting components in the RX and TX lane and the RF transformator) then the was making with the EFR32MG21 module.

I was hoping the they was updating the EFR32MG21 module so its being stable and also the main PCB that was having many hardware failure.

I can being wrong or the photo looks bad, is the Zigbee module "over cooked".

**Reply by Hedda** [reply]
> the SM-031 looks much better in the RF part (counting components in the RX and TX lane and the RF transformator) then the was making with the EFR32MG21 module.
>
> I was hoping the they was updating the EFR32MG21 module so its being stable and also the main PCB that was having many hardware failure.

We know that ITead outsourced the electrical engineering of "SM-011 V1.0" radio-module used in the original [Sonoff ZBBridge](https://itead.cc/product/sonoff-zbbridge/) as well as the design of their first "[Zigbee 3.0 USB Dongle](https://itead.cc/product/zigbee-3-0-usb-dongle/)" based on Silicon Labs based EFR32MG21 to [CoolKit-Technologies / Shenzhen Cool House Technology Co., Ltd. (abbreviated as Cool House Technology)](https://www.coolkit.cn/).

Looking at the board pictures of "[ITead SONOFF Zigbee 3.0 USB Dongle Plus"](https://itead.cc/product/sonoff-zigbee-3-0-usb-dongle-plus/) from @notenoughtech my guess is it is the same engineer at @CoolKit-Technologies who engineered this “SM-031 v1.1” radio-module that helped ITead design the board for the SONOFF Zigbee 3.0 USB Dongle Plus adapter:

https://notenoughtech.com/home-automation/sonoff-zigbee-3-0-usb-dongle-plus/

![image](https://user-images.githubusercontent.com/6320001/149287172-16030f55-0670-4189-a21c-97444fe2d4b8.png)

![image](https://user-images.githubusercontent.com/6320001/149287217-2a674124-8d24-4582-82f9-026b3da9abf3.png)

Will keep an eye out if [CoolKit ](https://www.coolkit.cn/)also start to offer this new "SM-031 v1.1” radio-module to third-parties as done with "SM-011 V1.0":

https://www.coolkit.cn/product/sm-011/

https://www.coolkit.cn/product/sm-031/

https://github.com/CoolKit-Technologies/DevDocs/tree/master/Zigbee

The "SM-031 v1.1” radio modules are sold on Alibaba and Aliexpress as well as being used in other gateways like eWeLink ZB-GW03:

https://github.com/zigpy/zigpy/discussions/819

https://github.com/zigpy/zigpy/discussions/586

> I can being wrong or the photo looks bad, is the Zigbee module "over cooked".

It could be so but it usually just early pre-production models that is sent into FCC for approval, so not production-line quality yet.

**Comment by Hedda** [comment]
FYI, ITead has apparently now started selling "**SONOFF Zigbee Bridge Pro**" which I assume are the same as the previously leaked "Sonoff ZBBridge-P" model based on CC2652P and ESP32-D0WD-V3?

https://itead.cc/product/sonoff-zigbee-bridge-pro/

![image](https://user-images.githubusercontent.com/6320001/166225949-8c9f767a-768e-4abc-8256-9af358715993.png)

Its packaging is the same as and their previous Zigbee gateway/bridge and Zigbee gateway/bridge and the only listed difference mentioned on its website is that it can handle 128 instead of 32 devices, contains a “buzzer” which webpage mention that the buzzer can be used as an alarm (siren?) or as a doorbell chime (if use Zigbee wireless buttons as doorbells), and a “built-in high-precision local RTC” (Real-Time Clock with clock-battery) for gateway time synchronization of local scene schedules.

![image](https://user-images.githubusercontent.com/6320001/166228468-2a9d9eac-3e81-44c0-a342-770ce6c6e1b7.png)

Note that they are also still selling the older ZBBridge based on EFR32MG21 and ESP8266/ESP8285 which has already been hacked with Tasmota and ESPHome as a network-attached Zigbee adapter:

https://itead.cc/product/sonoff-zbbridge/

Note that Zigbee2Tasmota (tasmota-zbbridge) is by the way already supported on Tube's Zigbee Gateways with similar specifications:

https://www.tubeszb.com/product/cc2652_coordinator/39

https://www.tubeszb.com/product/cc2652_poe_coordinator/47

https://github.com/tube0013/tube_gateways

(Tube's Zigbee Gateways are based on ESP32 dev board (Wireless-Tag WT32-ETH01 or Olimex ESP32-POE-ISO) combined with either a Texas Instruments CC2652P based Zigbee module or a Silicon Labs EFR32MG21 based Zigbee module).
